src/ekka_rlog_bootstrapper.erl

```diff
@@ -0,0 +1,131 @@
+%%--------------------------------------------------------------------
```

Comment:
Note: currently this module is not implemented. It just does minimal amount of signalling to let the replicant state machine progress to the normal state.